### PR TITLE
`dl-v2` cli improvements

### PR DESCRIPTION
**Pull request by aniketinfocusp**
This PR incldudes:
- now cli can be used from any directory as long as conda environment is on.
- empty queue options
- updating cli from cli itself
- minor fixes
- sorted imports of cli

### CODE REVIEW

File: weather_dl_v2/cli/app/cli_config.py [hunk 1]
```diff
@@ -12,11 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+import pkg_resources
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/cli_config.py:15]: Lets maintain lexicographical ordering

File: weather_dl_v2/cli/app/subcommands/queue.py [hunk 2]
```diff
@@ -59,6 +59,21 @@ def get_license_queue(license: Annotated[str, typer.Argument(help="License ID")]
...
+        bool,
+        typer.Option(
+            "--empty",
+            help="""Empties the license queue. If this is passed, other options are ignored."""
+        )
+    ] = False,
+    save: Annotated[
+        str,
+        typer.Option(
+            "--save-and-empty",
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/subcommands/queue.py:72]: Lets make it `_`.

File: weather_dl_v2/cli/app/subcommands/queue.py [hunk 3]
```diff
@@ -59,6 +59,21 @@ def get_license_queue(license: Annotated[str, typer.Argument(help="License ID")]
...
 def modify_license_queue(
     license: Annotated[str, typer.Argument(help="License ID.")],
+    empty: Annotated[
+        bool,
+        typer.Option(
+            "--empty",
+            help="""Empties the license queue. If this is passed, other options are ignored."""
+        )
+    ] = False,
+    save: Annotated[
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/subcommands/queue.py:69]: nit: save_dir_path.

File: weather_dl_v2/cli/app/subcommands/queue.py [hunk 4]
```diff
@@ -76,10 +91,27 @@ def modify_license_queue(
...
             "-p",
             help="Absolute priority for the config in a license queue."
-            "Priority increases in ascending order with 0 having highest priority.",
+            " Priority increases in ascending order with 0 having highest priority.",
         ),
     ] = None,  # noqa
 ):
+
+    if empty and save:
+        print("Both --empty and --save-empty can't be passed. Use only one.")
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/subcommands/queue.py:100]: --save_and_empty

File: weather_dl_v2/cli/app/cli_config.py (hunk 1 above)
Comment (aniketinfocusp) [reply weather_dl_v2/cli/app/cli_config.py]: resolved in [6cf31cf](https://github.com/google/weather-tools/pull/422/commits/6cf31cf2866762730543a057927df3c1283ab1a0)

File: weather_dl_v2/cli/app/subcommands/queue.py (hunk 3 above)
Comment (aniketinfocusp) [reply weather_dl_v2/cli/app/subcommands/queue.py]: done

File: weather_dl_v2/cli/app/utils.py [hunk 5]
```diff
@@ -14,15 +14,15 @@
...
-import typing as t
 import json
-from time import time
+import logging
+import typing as t
 from itertools import cycle
 from shutil import get_terminal_size
 from threading import Thread
-from time import sleep
+from time import sleep, time
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/utils.py:24]: ?

**Review by mahrsee1997 — APPROVED** [review]
LGTM.

**Pull request merged**